client/app/components/lesson-item.tsx

```diff
@@ -132,15 +145,29 @@ function LessonItem({ checked, lesson, onClick }: Props) {
                   <span>End time</span>
                   <Input
                     type="time"
-                    {...register("timeEnd", { required: true })}
+                    {...register("timeEnd", {
+                      required: true,
+                      validate: (value) =>
+                        isStartTimeBeforeEndTime(watch("timeStart"), value) ||
+                        "End time must be after start time",
+                    })}
                   />
+                  {errors.timeEnd && (
+                    <span className="text-red-500">
+                      {errors.timeEnd.message}
+                    </span>
+                  )}
```

Comment:
I haven't ran this locally but I'll assume, this crams the error message in that small space.

Can we move this so its below the _flex_ div that contains the three inputs?

Reply:
FWIW this is how it looks now, before moving per your suggestion. Doesn't look horrible to me

<img width="1349" alt="Screenshot 2024-01-28 at 15 16 43" src="https://github.com/blackmann/compa/assets/68050609/904d656d-082a-49c5-a020-42d9d78c78e9">

And this is how it looks below the 3 inputs:

<img width="1344" alt="Screenshot 2024-01-28 at 15 19 10" src="https://github.com/blackmann/compa/assets/68050609/f275beb1-bcc9-4d79-9b19-4a92f323e372">


Your choice :)

Reply:
You're right.

Let's try something; let's use <small /> and change the text to "should be after start time" so that it takes fewer space.